bin/cron/report_activejob_metrics

```diff
@@ -29,6 +29,15 @@ def push_queue_length_metrics
         dimensions: [
           {name: 'Environment', value: CDO.rack_env},
         ],
+      },
+      {
+        metric_name: 'WorkableJobCount',
+        value: Delayed::Job.where(failed_at: nil, locked_at: nil).where('run_at <= ?', Time.now).count,
+        unit: 'Count',
+        timestamp: Time.now,
+        dimensions: [
+          {name: 'Environment', value: CDO.rack_env},
+        ],
```

Comment:
not blocking for this PR: as this list grows, just wondering if it might start to be worth sharing code? either my accessing `concerns/active_job_metrics.rb` directly, or extracting it out  to dashboard/lib?

Reply:
Yeah, I agree. There's several ways to do that, I'm curious what you think the most railsish or aligned-to-our-codebase approach would be?

Reply:
I think dashboard/lib is probably the right solution, since there is community consensus that business logic not directly tied to a rails model, view or controller can/should go there. but I also think we are sorely lacking an accepted directory structure there which would make it obvious where to put stuff, which makes me hesitate. So I'd be inclined to just reach into this rails concern directly for now, but extracting to dashboard/lib could also be an option if you have a vision (or not and we can move stuff around later).

sorry for the non-answer. either seems fine 🤷

Reply:
another idea would be to put it in a rails helper, which might feel slightly less icky than having a cronjob reach into a concern.